mlflow/models/evaluation/default_evaluator.py

```diff
@@ -847,7 +857,7 @@ def _log_custom_metric_artifact(self, artifact_name, raw_artifact, custom_metric
         artifact._load(artifact_file_local_path)
         return artifact
 
-    def _evaluate_custom_metrics_and_log_produced_artifacts(self):
+    def _evaluate_custom_metrics_and_log_produced_artifacts(self, disable_logging=False):
```

Comment:
Nit: Can we switch the boolean to `log_to_mlflow_tracking=True` by default? Makes it clearer and removes the double negative.